pkg/odo/cli/preference/set.go

```diff
@@ -30,6 +30,7 @@ var (
    %[1]s %[6]s 30
    %[1]s %[7]s true
    %[1]s %[8]s docker
+   %[1]s %[9]s true
```

Comment:
I don't exactly like this kind of formatting because if the place of any preference changes [here](https://github.com/openshift/odo/pull/4456/files#diff-520872630a47e18df6266f58252a6fd10fe43eba71f49d23d346b2a76ec1b3baR99) without the change being made in the template, it can cause a misunderstanding in the help. I did not find an elegant enough solution but I would like if we can do something about it.

Reply:
You mean if someone changes the order of
```go
preference.ExperimentalSetting, preference.PushTargetSetting, preference.ConsentTelemetrySetting
```
to 
```go
preference.PushTargetSetting, preference.ConsentTelemetrySetting, preference.ExperimentalSetting
```
but doesn't make a change in this template, the `odo preference set -h` would mean something entirely different?

I think that's a valid point. But what's the solution? One way it could be made less confusing is by adding code comments.

Reply:
Yes, exactly that! I was hoping to find a solution more on the line where we could modify the template. I'll add a TODO for now.

Reply:
I would suggest to use package text/template.
That way you can write something like below e.g.

```

{{ .PushTargetSetting }} {{ .PushTargetSettingDefault }}

```

and this is quite descriptive and non positional.

you can consider looping over these values to make it even more cooler?

Reply:
there is ktemplate package which we use at some places? Does that work similarly? Check that too

Reply:
Currently, this piece of code is using ktemplates. But I do like the idea of text/template, it's exactly what I was looking for. I'll see if we can do something like that with ktemplate, if not, I'll go ahead with text/template.

Reply:
I raised another [PR](https://github.com/openshift/odo/pull/4471/commits/99b535e8cf7e461078646dd98e267f1d9ddf20b5) because this change seemed somewhat out of scope for this PR. I realized there might be other places requiring the same kind of change, so I plan on adding them later on to the same PR.